packages/mobile/src/app/AppContextProvider.tsx

```diff
@@ -24,6 +25,9 @@ export const AppContextProvider = (props: AppContextProviderProps) => {
       analytics,
       storageNodeSelector,
       localStorage,
+      getHostUrl: () => {
```

Comment:
feels dangerous to have this inferred on web and from env vars on mobile. maybe it can always be from env vars and we don't need something in app context for it?

Reply:
I tend to agree, but I'm not sure why it was split in the first place. I just wanted to preserve the logic in case it was meant to handle a specific edge case.
I'll investigate a little more and/or make a ticket to remove it.

Reply:
yeah that's fair.... but i guess mobile has no elegant solve to this